### PR TITLE
AIRNEIS-58-category-WEB

### PR DESCRIPTION
**Pull request by hallycsse**
category page :
image

Only front, as seen with @KyDev25 api/filter will be done later.

### CODE REVIEW

File: front/src/pages/categories/[categoryId].jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+import Page from "@/components/Page"
+import ProductCard from "@/components/ProductCard"
+
+const category = {
+  description: "LES SALONS ÉCOSSAIS OFFRENT UNE POSSIBILITÉ INFINIE À VOTRE MAISON. DÉCOUVREZ LA DOUCEUR ET LA FRAÎCHEUR DE VOTRE FUTUR SALON."
```
Comment (cedricgautier) [inline front/src/pages/categories/[categoryId].jsx:6]: Can you make a constants file for this

**Comment by cedricgautier** [comment]
Can you specify the meaning of only  having only front ?

File: front/src/pages/categories/[categoryId].jsx (hunk 1 above)
Comment (hallycsse) [reply front/src/pages/categories/[categoryId].jsx:6]: Only here to be replaced in the futur by the api call so does i really need to make a file for it.

File: front/src/components/Header.jsx [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
-          <Bars3Icon {...webConfig.icons.xlIcon} />
-        </li>
-      </ul>
-    </nav>
+const Header = ({ page }) => (
+  <header className="bg-secondary text-primary px-4 flex justify-between items-center sticky top-0 z-30">
+    <Link href="/" className="text-xl md:text-4xl py-3 font-extrabold">
+      ÀIRNEIS
+    </Link>
+    {page !== "backoffice" ? (
```
Comment (cedricgautier) [inline front/src/components/Header.jsx]: magic number, please make constant

**Comment by hallycsse** [comment]
only front signifying here than the page does not work with api call and it will be an other ticket work

File: front/src/components/Header.jsx (hunk 2 above)
Comment (hallycsse) [reply front/src/components/Header.jsx]: py-3 and text-4xl ?

File: front/src/pages/categories/[categoryId].jsx (hunk 1 above)
Comment (cedricgautier) [reply front/src/pages/categories/[categoryId].jsx:6]: Okay sounds good, thanks

File: front/src/components/Header.jsx (hunk 2 above)
Comment (cedricgautier) [reply front/src/components/Header.jsx]: "backoffice"

File: front/src/components/Header.jsx [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
-        </li>
-        <li>
-          {/* As this icon is smaller that the other we need it to have a bigger size */}
-          <Bars3Icon {...webConfig.icons.xlIcon} />
-        </li>
-      </ul>
-    </nav>
+const Header = ({ page }) => (
+  <header className="bg-secondary text-primary px-4 flex justify-between items-center sticky top-0 z-30">
+    <Link href="/" className="text-xl md:text-4xl py-3 font-extrabold">
```
Comment (cedricgautier) [inline front/src/components/Header.jsx]: Can you make a routes file and add the Home directory, thanks

**Review by cedricgautier — APPROVED** [review]
Few changes, but looks good to me.

File: front/src/components/ProductCard.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import Image from "next/image"
+
+const ProductCard = ({ product }) => (
+  <li className="flex flex-col gap-1">
+    <Image alt="#" src={product.image} width={450} height={200} className="rounded-sm drop-shadow-box" />
```
Comment (barnabepilliaudin) [inline front/src/components/ProductCard.jsx]: Can you put product name as description for SEO purpose

File: front/src/pages/categories/[categoryId].jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  const { categoryId } = query
+
+  return { props: { categoryId } }
+}
+const CategoryPage = props => {
+  const { categoryId } = props
+
+  return (
+    <Page>
+      <button className="w-full h-1/4 sm:h-1/3 bg-[url('https://picsum.photos/1800/350')] bg-center flex items-center justify-center">
```
Comment (barnabepilliaudin) [inline front/src/pages/categories/[categoryId].jsx]: Why having a Button here ?

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]

File: front/src/components/ProductCard.jsx (hunk 4 above)
Comment (hallycsse) [reply front/src/components/ProductCard.jsx]: I don't understand what u mean by product name as description can u explain it pls

Comment (barnabepilliaudin) [reply front/src/components/ProductCard.jsx]: Explained via discord DM

Comment (Magali-D) [reply front/src/components/ProductCard.jsx]: Is the change satisfying?

**Review by BarnabePILLIAUDIN — APPROVED** [review]

**Pull request merged**